license

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2017, Mapbox
```

Comment:
This repository's history stretches [back to 2014](https://github.com/mapbox/mapbox-android-demo/commit/b5de11688e06e65283321abfabbdba2a1fd09706). Presumably we should claim copyright for all the years since then.